package.json

```diff
@@ -60,7 +59,7 @@
   "devDependencies": {
     "@sveltejs/svelte-repl": "0.0.5",
     "@types/mocha": "^5.2.0",
-    "@types/node": "^10.5.5",
+    "@types/node": "=8",
```

Comment:
Does `=8` mean anything different than just `8`? Or `^8`?

Reply:
It is the same as `^8`, but makes it clear we don't intend on upgrading.